src/locales/strings.js

```diff
@@ -0,0 +1,109 @@
+// TODO(sjbarag): read from JS module generated from JSON files, probably.
+const translations = {
+    en: {
+        'hello': {
+            title: 'Hello world',
+            note: 'Static text for testing.'
+        },
+        'lipsum': {
+            title: 'Lorem ipsum dolor sit amet, {foo} {bar}',
+            note: 'Placeholder text.'
+        },
+        'usePersonalDuckAddr': {
+            title: 'Use {email}',
+            note: 'Shown when a user can choose their personal @duck.com address.'
```

Comment:
@sjbarag These notes are for translators, so we should try to provide the context necessary for them to provide the best possible translation. Here's a better example.

```suggestion
            note: 'Button that fills a form using a specific email address. The placeholder is the email address. E.g. "Use test@duck.com".'
```